docs/EndToEndTests/Settings.md

```diff
@@ -0,0 +1,56 @@
+## Settings
+
+- [Settings](#settings)
+  - [Overview](#overview)
```

Comment:
this section seems to be missing in this document

Reply:
Good catch! I think I copied over it accidentally when I added the table of contents.